plugin/src/cli.ts

```diff
@@ -63,9 +101,20 @@ async function run() {
   // This value has been tested and gives best peformance for most scenarios.
   if (!GATSBY_CPU_COUNT && process.env.NETLIFY && !process.env.NETLIFY_LOCAL) {
     const reportedCores = cpuCoreCount(true)
-    GATSBY_CPU_COUNT = Math.max(reportedCores - 2, 2).toString()
+    const threads = os.cpus().length
+    const threadsPerCore = threads / reportedCores
+
+    let cpuAllocation = getCpuAllocation()
+    cpuAllocation ||= threads
```

Comment:
[dust] I think this could also be written as 
```suggestion
    const cpuAllocation = getCpuAllocation() || threads
```

Reply:
Haha. Thanks. Not sure what I was thinking there.